src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Email/Blocks/email_configuration.html.twig

```diff
@@ -25,7 +25,7 @@
 {% trans_default_domain 'Admin.Advparameters.Feature' %}
 
 {% block email_configuration %}
-  <div class="col-xl-10">
+  <div class="col-12">
```

Comment:
Isn't `col` enough for using the whole width?

Reply:
i've updated with `col` where possible. If you use 2 or more `col` in same row, they wont take up all width.